submissions/madmaxWMFU/html-movie-booking/index.html

```diff
@@ -0,0 +1,894 @@
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-5" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-5" name="seat" data-value="2-5">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-6" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-6" name="seat" data-value="2-6" checked="">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-7" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-7" name="seat" data-value="2-7" checked="">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-8" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-8" name="seat" data-value="2-8" >
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-9" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-9" name="seat" data-value="2-9">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-10" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-10" name="seat" data-value="2-10">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-11" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-11" name="seat" data-value="2-11">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-12" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-12" name="seat" data-value="2-12">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-13" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-13" name="seat" data-value="2-13">
+                            <span class="hall__seat-inner ordinary"></span>
+                        </label>
+                        <label for="seat2-14" class="hall__seat">
+                            <input class="hall__seat-data" type="checkbox" id="seat2-14" name="seat" data-value="2-14">
```

Comment:
Same styles for checked and unavailable seats isn't a good UX decision.
Also, add `cursor: pointer` to seat.